reflex/experimental/__init__.py

```diff
@@ -43,7 +42,6 @@ def toast(self):
     client_state=ClientStateVar.create,
     hooks=hooks,
     layout=layout,
-    progress=progress,
```

Comment:
can we add this to the ExperimentalNamespace with a promotion warning in case anyone was already using it

Reply:
should we have/specify a removal version for promoted experimental features/components just like we do for deprecated features?

Reply:
it's more of a courtesy really. anything in `rx._x` can be removed or changed at any time